content/collections/data/en/automated-tasks-in-data-assistant.md

```diff
@@ -0,0 +1,100 @@
+---
+id: bef1ff7c-2c93-4677-b05e-1588205fbd4d
+blueprint: data
+title: 'Automated Tasks'
+this_article_will_help_you:
+  - 'Understand how automated tasks can help keep your data clean'
+  - 'Learn how to set up and run automated tasks'
+landing: false
+exclude_from_sitemap: false
+updated_by: 3f7c2286-b7da-4443-a04f-7c225af40254
+updated_at: 1754950505
+---
+Automated Tasks are part of Amplitude’s [Data Assistant](/docs/data/use-ai-data-assistant) feature. Automated tasks let data governors define conditions and actions that the system then performs automatically. It automates repetitive clean-up workflows by taking recommendations provided by the Data Assistant and transforming those recommendations into automatic actions. For example, an automated task can automatically delete events if they have been unused for 90 days. 
```

Comment:
"Automated Tasks" (first sentence) vs. "Automated tasks" (second sentence)

Third sentence starts "It" but previous mentions sound as though Automated Tasks should be plural.

Third sentence sounds passive. "Automated tasks use the recommendations provided from Data assistant to automate repetitive tasks" or something.